.github/workflows/raindrops-ci.yml

```diff
@@ -0,0 +1,56 @@
+name: Raindrops CI
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Install Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: "yarn"
+          cache-dependency-path: ./yarn.lock
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          components: clippy
+
+      - name: Install Solana
+        run: curl -sSfL https://release.solana.com/stable/install | sh
+
+      - name: Install Anchor Dependencies
+        run: sudo apt-get update && sudo apt-get upgrade && sudo apt-get install -y pkg-config build-essential libudev-dev
+
+      - name: Install Anchor 0.24.2
+        run: npm i -g @project-serum/anchor-cli@0.24.2
+
+      - name: Set Environment
+        run: echo "/home/runner/.cargo/bin" >> $GITHUB_PATH && echo "/home/runner/.local/share/solana/install/active_release/bin" >> $GITHUB_PATH
+
+      - name: Install Node.js dependencies
+        run: yarn install --immutable
+
+      - name: Lint
+        run: yarn run lint:check
+
+      - name: Build
+        run: yarn run build:lint
+
+      - name: Test
+        run: yarn run test
```

Comment:
Can we add support for the program tests since the system tests aren't merged yet?

Reply:
It already supports both program tests and system wide tests.
Because root `yarn run test` runs every test script in workspaces